docs/tutorial/code/node-app/task.yaml

```diff
@@ -38,6 +38,10 @@ execute: |
   # Wait for a bit for the app to come online
   retry -n 5 --wait 2 curl localhost:8000
 
+  # [docs:curl-localhost]
+  curl localhost:8000
```

Comment:
To match the tutorial exactly, this should include the http scheme.

Reply:
thanks for pointing this out, I meant to include this bit in the tutorial file!
updated